x-pack/test/security_solution_api_integration/test_suites/entity_analytics/risk_engine/trial_license_complete_tier/init_and_status_apis.ts

```diff
@@ -539,6 +539,86 @@ export default ({ getService }: FtrProviderContext) => {
           firstResponse?.saved_objects?.[0]?.id
         );
       });
+
+      it('should update the existing component template and index template without any errors', async () => {
```

Comment:
Just to confirm, does this test fail without the fix?

Reply:
Yes, this test fails without the fix. I checked before making the change and it failed with the same error that we see on the API or UI.